zenohd/src/main.rs

```diff
@@ -120,7 +120,11 @@ fn config_from_args(args: &Args) -> Config {
         .config
         .as_ref()
         .map_or_else(Config::default, |conf_file| {
-            Config::from_file(conf_file).unwrap()
+            Config::from_file(conf_file).unwrap_or_else(|e| {
+                // if file load fail, wanning it, and load default config
+                tracing::warn!("Warn: File {} not found! {}", conf_file, e.to_string());
+                Config::default()
```

Comment:
If config file is provided as CLI arg and the file does not exist, I believe we should just return an error and exit. I don't think it's a good a idea to silently fallback to a default configuration here.

Reply:
Not only `DEFAULT_CONFIG.json5`, I believe zenohd should refuse to start if wrong parameters are passed and not to silently fallback. The user may not even see the traficing warn if zenohd is run in a container. But it's very apparent if zenohd does not start at all.

Reply:
Reverted the fix, reopened issue https://github.com/eclipse-zenoh/zenoh/issues/1292